browserstack.conf.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+	bs_chrome: {
```

Comment:
Looks like these are repeated in 3 places. Might it make sense to pull these out into their own file? I'm fine either way honestly.